firebase-vscode/package.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "firebase-vscode",
-  "displayName": "firebase-vscode",
+  "displayName": "Firebase Extension",
```

Comment:
Prefer any other name to avoid clashing with the separate product Firebase Extensions. How about just Firebase?